src/main/java/org/codehaus/groovy/ast/expr/RangeExpression.java

```diff
@@ -27,26 +27,42 @@
  * <pre>for i in 0..10 {...}</pre>
  */
 public class RangeExpression extends Expression {
-
     private final Expression from;
     private final Expression to;
-    private final boolean inclusive;
+    private final boolean inclusive; // Kept to keep old code depending on this working
```

Comment:
Can this field be dropped and `isInclusive()` is implemented in terms of "right inclusivity"?

Reply:
Agreed, I'll implement this.